src/services/AffectService.ts

```diff
@@ -35,3 +35,49 @@ export async function deleteAffect(uuid: string) {
     .then(createSuccessHandler({ title: 'Success!', body: 'Affect Deleted.' }))
     .catch(createCatchHandler('Error deleting Affect:'));
 }
+
+
+
+// {
+// "comment": "string",
+// "cvss_version": "V2",
+// "issuer": "RH",
+// "vector": "string",
+// "embargoed": true
+//   "updated_dt": "2024-02-06T16:02:54.708Z"
+// }
+export async function putAffectCvssScore(
+  affectId: string,
+  cvssScoresId: string,
+  cvssScoreObject: unknown,
+) {
```

Comment:
Why having `cvssScoreObject ` as unknown?
(same for line 74)

Reply:
It's basically a placeholder type, but I can declare these as actual types using the 'schema' in the comments above.

Reply:
No preference from me